codebuild/spec/buildspec_ubuntu_integrationv2.yml

```diff
@@ -29,7 +29,7 @@ batch:
         variables:
           S2N_BUILD_PRESET:
             - awslc_gcc9
-            - awslc-fips_gcc4-8
```

Comment:
I think the valgrind buildspec should be updated as well:
https://github.com/aws/s2n-tls/blob/25b08ba9acd1745d85a10f56b8e1bd0a686a1683/codebuild/spec/buildspec_valgrind.yml#L30